### DISCUSSION TITLE
Can I format one column to be as text

### DISCUSSION THREAD

**Discussion by JanneHarju** [mini-software/MiniExcel#645] [Q&A]
I'm trying to generate empty file for being template for user to insert data in it and then uploading it. Few columns are strings but they might contain only numbers like 000456546. If I download this template file new rows which I add in Excel are treated as numbers. So if user tries to add row where reference number is like 0034535, 00 are removed. User need to format whole column to Text to get over this problem. Can I format whole column to Text? Here is my example code of it. MyDto contains String, number and DateTime fields.

>   var memoryStream = new MemoryStream();
      var exampleContent = new List<MyDto>();
      await MiniExcel.SaveAsAsync(memoryStream, exampleContent);
      memoryStream.Seek(0, SeekOrigin.Begin);
      var filename = "ExampleTemplate.xlsx";
      var result = new RemoteStreamContent(memoryStream, filename, "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet");
      return await Task.FromResult(result);

**Comment by michelebastione** [comment]
This is not currently supported.
